common/djangoapps/student/views.py

```diff
@@ -1582,6 +1582,10 @@ def _do_create_account(form, custom_form=None):
                 custom_model = custom_form.save(commit=False)
                 custom_model.user = user
                 custom_model.save()
+
+            if site:
+                # Set UserAttribute indicating the site the user account was created on.
+                UserAttribute.set_user_attribute(user, 'created_on_site', site.domain)
```

Comment:
This is specifically why I'm recording the site domain here. `site.domain` can change. I want this attribute to capture the domain of the site that the account was created on.